LSP.sublime-settings

```diff
@@ -163,8 +163,9 @@
   // sorting algorithm and instead uses the sorting defined by the relevant language server.
   "inhibit_word_completions": true,
 
-  // Controls whether words are overwritten when accepting completions.
-  // Note that a language server needs to support this feature to work.
+  // For servers that support it, chooses the primary mode used for inserting completions.
+  // For example, when triggering completion in a middle of a word, `insert` would insert the completion text in a middle of the word, while `replace` would replace the existing word with a new completion text.
+  // An LSP keybinding `lsp_commit_completion_with_opposite_insert_mode` can be used to insert completion using the opposite mode to the one selected here.
```

Comment:
Very long lines. Can you make them more close to convention in this file?

Reply:
Will try 🙂